### PR TITLE
add radcli 1.0.0 [rpm]

### PR DESCRIPTION
**Pull request by timogoebel**
Packaging for [radcli](https://rubygems.org/gems/radcli), a dependency for [smart_proxy_realm_ad_plugin](https://rubygems.org/gems/smart_proxy_realm_ad_plugin). Please see https://github.com/theforeman/smart-proxy/pull/557 for discussion.

Please build for:

* [x] Nightly
* [x] 1.16
* [x] 1.15
* [ ] 1.14

### CODE REVIEW

**Comment by timogoebel** [comment]
[test]

File: rubygem-radcli/rubygem-radcli-0.1.0.spec [hunk 1]
```diff
@@ -0,0 +1,84 @@
+%global gem_name radcli
+
+Name: rubygem-%{gem_name}
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: A Ruby interface for the adcli library
+Group: Development/Languages
+License: Artistic-2.0
+URL: http://github.com/martencassel/radcli
```
Comment (ekohl) [inline rubygem-radcli/rubygem-radcli-0.1.0.spec]: Can be https

**Comment by timogoebel** [comment]
@ekohl: Changed to https. Note to self: We should also fix that in the gemspec.

**Comment by ekohl** [comment]
https://github.com/martencassel/radcli/pull/5

**Comment by ekohl** [comment]
Yesterday I tagged it into nightly, now also 1.16 and 1.15. I forgot to tag it into f24 but that shouldn't be a problem.

**Pull request merged**